app/javascript/packs/modals/unpublishAllPosts.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable no-restricted-globals */
```

Comment:
Is this needed? I tried removing it locally and eslint didn't throw up any errors but maybe I'm missing something